packages/design-system-twrnc-preset/package.json

```diff
@@ -46,7 +46,7 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
-    "@metamask/design-tokens": "workspace:^"
+    "twrnc": "^4.5.1"
   },
```

Comment:
We include `twrnc` in dependencies to ensure Tailwind classnames are accessed exclusively via our `useTailwind` hook and `design-system-twrnc-provider`. This enforces the use of our classnames and prevents direct usage of `twrnc` elsewhere in the project.